docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+ path through the decision tree, :math:`p(\text{path})` is the probability of that path; for example the path that includes the edges labeled BEmONC and 
+ unavailable occurs with probability that the birth is in a BEmONC facility times the probability that the facility has azithromycin available.
+
+When we fill in the location-specific values for delivery facility rates, azithromycin coverage, relative risk of sepsis incidence with azithromycin access, 
+and sepsis incidence probability (which is also age-specific), this becomes a system of two linear equations with two unknowns (:math:`p(\text{sepsis}|\text{azithromycin})` 
+and :math:`p(\text{sepsis}|\text{no azithromycin})`), which we can solve analytically using the same approach as in the :ref:`cpap calibration <cpap_calibration>`.
+
+**Alternative PAF Derivation**: An alternative, and possibly simpler derivation of the PAF that will calibrate this model comes from the observation that
+:math:`\text{PAF} = 1 - \frac{1}{\mathbb{E}(\text{RR})}`.  If we define 
+
+.. math::
+
+   p(\text{no azithromycin}) = \sum_{\text{paths without azithromycin}} p(\text{path}),
+
+then can use this to expand the identity
+
+.. math::
+
+   \text{PAF}_\text{no azithromycin} = 1 - \frac{1}{\mathbb{E}(\text{RR})}.
+
+Since our risk exposure has two categories,
+
+.. math::
+
+   \mathbb{E}(\text{RR}) = p(\text{no azithromycin}) \cdot \text{RR}_\text{no azithromycin} + (1 - p(\text{no azithromycin})) \cdot 1.
+
+
+
+Scenarios
+---------
+
+.. todo::
+
+  Describe our general approach to scenarios, for example set coverage to different levels in different types of health facilities; then the specific values for specific scenarios will be specified in the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+
+Assumptions and Limitations
+---------------------------
+
+- We assume that azithromycin availability captures actual use, and not simply the treatment being in the facility. 
+- We assume that the delivery facility is also the facility where a mother or birthing person will seek care for sepsis.
+- We assume that the relative risk of sepsis incidence with azithromycin in practice is a value that we can find in the literature (Note: 
+  the value we are using is from `this 2024 systematic review <https://bmcpregnancychildbirth.biomedcentral.com/articles/10.1186/s12884-024-06390-6#:~:text=Primary%20outcomes,-Among%20the%20six&text=The%20incidence%20of%20maternal%20sepsis%20was%20significantly%20lower%20in%20the,was%20analysed%20in%20three%20studies.>`_)
+- We have excluded the effect of azithromycin on pneumonia incidence/mortality, because this cause is currently lumped with 'other causes'.
+- We currenty do not model the impact of intrapartum azithromycin on the incidence of preterm births, despite *some* literature
+  evidence that suggests there may be a significant impact. For reference, this `2021 systematic review <https://pmc.ncbi.nlm.nih.gov/articles/PMC8436060/>`_
```

Comment:
It looks like this is specific to azithromycin in pregnancy rather than in labor. I don't think we would expect intrapartum azithromycin to have any affect on these sorts of outcomes. However, looking back at the slides they gave us initially, it does look like they noted they are interested in azithromycin during pregnancy to treat STIs, so let's keep it as a note in case we want to model that too.